src/cdn.cls.php

```diff
@@ -306,8 +306,12 @@ private function _replace_inline_css()
 		 * @see  #685485
 		 * @since 3.0
 		 */
-		preg_match_all('/url\((?![\'"]?data)[\'"]?([^\)\'"\\\]+)[\'"]?\)/i', $this->content, $matches);
+		preg_match_all('/url\((?![\'"]?data)[\'"]?(.+?)[\'"]?\)/i', $this->content, $matches);
 		foreach ($matches[1] as $k => $url) {
+			if (!$url) {
```

Comment:
As its used + now, is this condition redundant? @timotei-litespeed

Reply:
removed